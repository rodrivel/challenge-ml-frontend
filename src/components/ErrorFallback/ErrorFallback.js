import React from 'react';

const ErrorFallback = (error) => {
  return (
    <div>
      <p>Ups, algo sali√≥ mal.</p>
      <pre>{error.message}</pre>      
    </div>
  );
};

export default ErrorFallback;