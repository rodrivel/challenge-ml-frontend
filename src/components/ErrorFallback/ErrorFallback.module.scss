@import '/src/styles/base';

.ErrorFallback {

  margin-top: 30px;

  .ErrorFallback__MessageWrapper {
    display: flex;
    align-items: center;

    h1 {    
      font-size: 24px;
      font-weight: 300;      
      color: $primary-text
    }

    svg {
      fill: $primary-text;
      margin-right: 5px;
    }
  }
  
}