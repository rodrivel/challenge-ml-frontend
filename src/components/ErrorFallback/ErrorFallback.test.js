import React from 'react';
import { shallow } from 'enzyme';
import ErrorFallback from './ErrorFallback';

it('expect to render ErrorFallback component', () => {
  expect(shallow(<ErrorFallback/>)).toMatchSnapshot();
});