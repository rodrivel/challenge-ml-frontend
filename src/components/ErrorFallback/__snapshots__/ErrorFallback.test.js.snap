// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render ErrorFallback component 1`] = `ShallowWrapper {}`;
